### DISCUSSION TITLE
Github markdown alerts render incorrectly on website

### DISCUSSION THREAD

**Discussion by overthemike** [pmndrs/valtio#991] [Bug report]
### Bug Description

A while back, I updated the docs to use markdown alerts (example below). On github, this renders correctly with an icon, but on the website, it renders with the syntax below.
> [!NOTE] This is an example

I'll jump in and fix this but wanted to ask if I should just make them regular indented quotes, like so:
> Note: This is an example

or if I should first try and see if I can make it work on the website as well.

### Reproduction Link

https://valtio.dev/docs/how-tos/how-to-organize-actions

**Comment by dai-shi** [comment]
I think if you use emoji char directly, it would work for both.

**Reply by overthemike** [reply]
I had actually already started doing that as a back up - sounds good. I'll push up shortly.
